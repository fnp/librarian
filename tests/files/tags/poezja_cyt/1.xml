<poezja_cyt>
<strofa>
Tymczasem przenoś duszę moją utęsknioną/
Do tych pagórków leśnych, do tych łąk zielonych...
</strofa>
</poezja_cyt>
