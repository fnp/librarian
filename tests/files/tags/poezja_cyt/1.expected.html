<div id="book-text"><blockquote><div class="stanza" id="i1">
<a href="#f1" class="wl-num">1</a><div class="wl verse" id="f1">Tymczasem przenoś duszę moją utęsknioną</div>
<div class="wl verse" id="f2">Do tych pagórków leśnych, do tych łąk zielonych...</div>
</div></blockquote></div>
